src/app_engine/app.yaml

```diff
@@ -55,10 +53,6 @@ libraries:
 env_variables:
   BYPASS_JOIN_CONFIRMATION: false
   # Only change these while developing, do not commit to source!
-  # Use appcfg.py --env_variable=CALLSTATS_APP_ID:ID  \
```

Comment:
We want to keep these as we do use them when we deploy AppRTC. This will save us from having internal documentation. But you are correct that gcloud does not have a way to do this hence why we still use appcfg to deploy. Gcloud is used everywhere else though.